.github/workflows/release.yml

```diff
@@ -18,36 +19,16 @@ jobs:
       - name: Build APK
         id: build
         run: bash ./gradlew assembleRelease
-
-      - name: Sign APK
-        id: sign_apk
-        uses: r0adkll/sign-android-release@v1
+       
+      - name: Upload a Build Artifact
+        uses: actions/upload-artifact@v3
         with:
-          releaseDirectory: app/build/outputs/apk/release
```

Comment:
Why?! APKs are signed through keys stored as repository secret and this file is used to build and release APK and AAB file using GitHub actions, why you've changed paths and removed references of signing keys? this workflow is not functional at all, Please revert this change.

Reply:
I built an debug apk for my project that i managed named AnyTech,This project for translate apps to arabic language.

Reply:
I will refork repo again and add arabic strings.xml file only

Reply:
Alright, Thanks!